src/mumble/Audio.h

```diff
@@ -17,6 +17,9 @@
 
 #define SAMPLE_RATE 48000
 
+// interaural delay (in samples) for a sound coming directly from the side of the head
+constexpr float INTERAURAL_DELAY = 0.00043 / (1 / static_cast< float >(SAMPLE_RATE));
```

Comment:
Is the `static_cast` required?

Reply:
I think so. `SAMPLE_RATE` is an integer literal and therefore `1 / SAMPLE_RATE` would perform an integer division yielding `0`. Therefore either the `1` or `SAMPLE_RATE` has to be cast to a floating point number.

Reply:
Should I change this to `1.0 / SAMPLE_RATE` to make it more clear?

Reply:
Personally I don't mind the cast. If anything it makes it very clear that there is a real intention behind using a float type here (which is a good thing) :thinking: :shrug: